node/tests/cli.rs

```diff
@@ -116,107 +127,78 @@ fn read_config_override_values() {
         ws_rpc_addr: Some(ws_rpc_addr),
         ws_rpc_enabled: Some(false),
     };
-    let datadir_opt = Some(data_dir.path().into());
-    let config = NodeConfigFile::read(&config_path, &datadir_opt, &options).unwrap();
-
-    assert_eq!(config.datadir, data_dir.path());
+    let config = NodeConfigFile::read(&config_path, &options).unwrap();
 
     assert_eq!(
-        config.chainstate.chainstate_config.max_db_commit_attempts,
+        config
+            .chainstate
+            .clone()
+            .unwrap_or_default()
+            .chainstate_config
+            .max_db_commit_attempts,
         Some(max_db_commit_attempts)
     );
     assert_eq!(
-        config.chainstate.chainstate_config.max_orphan_blocks,
+        config
+            .chainstate
+            .clone()
+            .unwrap_or_default()
+            .chainstate_config
+            .max_orphan_blocks,
         Some(max_orphan_blocks)
     );
     assert_eq!(
-        config.chainstate.chainstate_config.tx_index_enabled,
+        config.chainstate.clone().unwrap_or_default().chainstate_config.tx_index_enabled,
```

Comment:
Why `unwrap_or_default` instead of `unwrap` is used here?

Reply:
Because this was the behavior before. But maybe I should change it back.